src/Symfony/Component/String/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+4.4.0
```

Comment:
5.0.0

Reply:
The dicsussion was [here](https://github.com/symfony/symfony/pull/33553#pullrequestreview-287270415), I don't know why it was resolved :thinking:

Reply:
5.0.0 updated (I resolved it when I rebased on master)